admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -210,6 +211,7 @@ export const ApplicationPage: React.FunctionComponent = () => {
                     </Tbody>
                 </TableComposable>
             </PageSection>
+            <EmailTemplateTable />
```

Comment:
Could we hide this for non admins?
on top:
` { isAdmin }  = usePermissions();`

here:
```suggestion
            { isAdmin && <EmailTemplateTable /> }
```